framework/include/auxkernels/FluxAux.h

```diff
@@ -0,0 +1,39 @@
+
+#ifndef FLUXAUX_H
+#define FLUXAUX_H
+
```

Comment:
You'll need the MOOSE headers at the top of all source and header files.  Copy from another file in this directory.